lib/content-services/src/lib/search/components/search-check-list/search-check-list.component.html

```diff
@@ -1,19 +1,20 @@
 <div class="checklist">
-    <mat-checkbox
-        *ngFor="let option of options"
-        [checked]="option.checked"
-        (keydown.enter)="option.checked = !option.checked"
-        [attr.data-automation-id]="'checkbox-' + (option.name)"
-        [attr.aria-label]="option.name | translate"
-        (change)="changeHandler($event, option)"
-        class="adf-facet-filter">
-        <div matTooltip="{{ option.name | translate }}"
-            matTooltipPosition="right"
-            class="facet-name">
-            {{ option.name | translate }}
-        </div>
-
-    </mat-checkbox>
+    <fieldset>
```

Comment:
Does `fieldset` require `legend` element as well?

Reply:
Indeed a legend is required with fieldset, which in this case should be 'Check Type'. But in this case the label is not being set in the template, but from search-header-query-builder service.